app/src/main/java/com/bogdan/codeforceswatcher/activity/AddUserActivity.kt

```diff
@@ -43,7 +43,7 @@ class AddUserActivity : AppCompatActivity(), OnClickListener {
                 if (response.isSuccessful) {
                     if (response.body()!!.result.firstOrNull() == null) {
                         progressBar.visibility = View.INVISIBLE
-                        showError()
+                        CwApp.app.showError()
```

Comment:
I'm not fan of such error showing, because you directly ping `CwApp` class, which shouldn't deal with any UI at all.